cmd/kyma/install/cmd.go

```diff
@@ -219,21 +235,33 @@ func (cmd *command) validateFlags() error {
 		if _, err := os.Stat(filepath.Join(cmd.opts.LocalSrcPath, "installation", "resources")); err != nil {
 			return fmt.Errorf("Configured 'src-path=%s' does not seem to point to a Kyma repository. Check if your repository contains the 'installation/resources' folder.", cmd.opts.LocalSrcPath)
 		}
+		break
 
-		// This is to help developer and use appropriate repository if PR image is provided
-		if cmd.opts.LocalInstallerDir == "" && strings.HasPrefix(cmd.opts.LocalInstallerVersion, "PR-") {
-			cmd.opts.LocalInstallerDir = "eu.gcr.io/kyma-project/pr"
-		}
-	} else {
-		if cmd.opts.LocalSrcPath != "" {
-			return fmt.Errorf("You specified 'src-path=%s' without specifying --local", cmd.opts.LocalSrcPath)
-		}
-		if cmd.opts.LocalInstallerVersion != "" {
-			return fmt.Errorf("You specified 'installer-version=%s' without specifying --local", cmd.opts.LocalInstallerVersion)
-		}
-		if cmd.opts.LocalInstallerDir != "" {
-			return fmt.Errorf("You specified 'installer-dir=%s' without specifying --local", cmd.opts.LocalInstallerDir)
+	//Install the latest version (latest master)
+	case strings.ToLower(cmd.opts.Source) == "latest":
+		latest, err := cmd.getMasterHash()
+		if err != nil {
+			return fmt.Errorf("Unable to get latest version of kyma: %s", err.Error())
 		}
+		cmd.opts.ReleaseVersion = fmt.Sprintf("master-%s", latest)
+		cmd.opts.ConfigVersion = "master"
+		cmd.opts.RegistryTemplate = registryMasterImagePattern
+		break
+
+	//Install the specific version from release (ex: 1.3.0)
+	case cmd.isSemVer(cmd.opts.Source):
+		cmd.opts.ReleaseVersion = cmd.opts.Source
+		cmd.opts.ConfigVersion = cmd.opts.Source
+		cmd.opts.RegistryTemplate = registryReleaseImagePattern
+		break
+
+	//Install the kyma with the specific installer image (docker image URL)
+	case cmd.isDockerImage(cmd.opts.Source):
+		cmd.opts.RemoteImage = cmd.opts.Source
+		cmd.opts.ConfigVersion = DefaultKymaVersion
+		break
+	default:
+		return fmt.Errorf("Source flag is not specified or it is not 'local' or a valid semver (ex: 1.3.0) or a docker image url")
```

Comment:
```suggestion
		return fmt.Errorf("Failed to parse the source flag. It can take one of the following: 'local', 'latest', release version (e.g. 1.4.1), or installer image")
```